app/src/main/res/values/strings.xml

```diff
@@ -33,9 +33,9 @@
   <string name="stop_exploration_dialog_description">Your progress will not be saved.</string>
   <string name="stop_exploration_dialog_cancel_button">Cancel</string>
   <string name="stop_exploration_dialog_leave_button">Leave</string>
-  <string name="topic_practice_master_these_skills">Master These Skills</string>
+  <string name="topic_practice_master_these_skills">Master these Skills</string>
```

Comment:
`Master These Skills`
https://xd.adobe.com/view/e8aa4198-3940-47f9-514a-f41cc54457f6-9e9b/screen/f403add1-079a-4202-9975-efe04c076290/TP-Practice-Unchecked-

Reply:
As per the mock. But I thought it needs to be corrected.

Reply:
Can you share that mock link here?

Reply:
> Can you share that mock link here?

No I was thinking there need to be correction or what. Reverted it back.